### PR TITLE
Support values refactor

### PR DESCRIPTION
**Pull request by sgibson91**
- Move support values or each cluster from the `cluster.yaml` file to a `support.values.yaml` file
- Adds `.helmignore`, `values.schema.yaml` to `support` chart
- Updates `cluster.schema.yaml`
- Refactors `deploy-support` and `deploy-grafana-dashbaords` to handle being passed files instead of straight values
- Adds a `validate_support_config` function when `deploy-support` is called

### CODE REVIEW

**Comment by consideRatio** [comment]
@sgibson91 i made a PR to this PR, see https://github.com/sgibson91/pilot-hubs/pull/135

**Comment by consideRatio** [comment]
I'd also want to see that we make sure that the CI system validates the support chart on PRs making changes.

**Comment by sgibson91** [comment]
> I'd also want to see that we make sure that the CI system validates the support chart on PRs making changes.

@consideRatio the last 2 commits I pushed should address this

**Comment by consideRatio** [comment]
@sgibson91 nice!

Hey btw, you may want to start using pre-commit locally to avoid needing to get pushed commits to your PRs. To do that, `pre-commit install --install-hooks` - and you will never make a commit that doesn't please pre-commit again in this repo :)

**Review by consideRatio — APPROVED** [review]
Wieeeeee nice this LGTM!

**Comment by consideRatio** [comment]
@sgibson91 it seems like RTD builds are triggered but failing to start. I suspect their CI service is down atm or experiencing troubles. This can be merged as it is without the RTD CI check in my mind though.

File: deployer/__main__.py [hunk 1]
```diff
@@ -355,6 +391,7 @@ def main():
         )
     elif args.action == "validate":
         validate_cluster_config(args.cluster_name)
+        validate_support_config(args.cluster_name)
```
Comment (choldgraf) [inline deployer/__main__.py:394]: Seeing these three `validate_` functions makes me happy :-)

**Comment by sgibson91** [comment]
I will check back in on the CI deployment after I've been to the gym 💪🏻

File: deployer/__main__.py (hunk 1 above)
Comment (sgibson91) [reply deployer/__main__.py:394]: Only useful if they work though 😉 https://github.com/2i2c-org/infrastructure/pull/1048#discussion_r817946077

Comment (consideRatio) [reply deployer/__main__.py:394]: @sgibson91 we just need `validate_validation_config()` as well :D #1049

**Pull request merged**